### ISSUE TITLE
When running TB.dev getting started, a user may think SCALARS and HPARAMS don't work

### ISSUE THREAD

**Issue by masoudcharkhabi** [tensorflow/tensorboard#4234]
When running TB.dev getting started [1], a user may think SCALARS and HPARAMS are not working because the messages that appear in the browser on the SCALARS and HPARAMS plugins are that "no scalar data is found" and "no hparams data is found". This is while the terminal says "uploader started" and the other plugins are working. A user has to refresh once the upload is complete but it would be better if the browser message was more clear. For large log uploads a user may think the SCALARS and HPARAMS plugins are not working, since the terminal message is not obvious.

![image](https://user-images.githubusercontent.com/7897145/96036437-4e908f00-0e19-11eb-9dc8-50d07b730aed.png)
![image](https://user-images.githubusercontent.com/7897145/96036532-767ff280-0e19-11eb-86be-e7cd6807db61.png)

**Comment by wchargin** [comment]
True. For context, local TensorBoard will only show panes for plugins
that have data available: if you have graphs and histograms but no
scalars, we won’t show the “no scalar data” message; instead, we’ll show
you the graphs and histograms.

Since TensorBoard.dev originally launched with only scalars, we just
made it always show “all” the dashboards (i.e., originally, just
scalars). But now would be a good time to properly implement
`DataProvider.list_plugins` based on what data’s available. Some
TBdev-specific messaging on the various “no data” copy could also be
helpful.
